.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,64 @@
+name: "\U0001F41E Bug report"
+description: Create a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before You Start...**
+
+        Thank you for taking the time to report a bug!
+        Please make sure to fill as much details as possible to help us better understand the bug.
+        If you have a question, suggestion, feature request, or need for help, please ask on [GitHub Discussions](https://github.com/mondaycom/monday-ui-react-core/discussions).
+        Thanks!
+  - type: input
+    id: version
+    attributes:
+      label: Vibe version
+      placeholder: e.g. 2.84.0
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce the bug
```

Comment:
Maybe consider changing the order of "Steps to reproduce the bug" with "A clear and concise description of what you expected to happen"